maintainers/maintainer-list.nix

```diff
@@ -8845,6 +8845,12 @@
     githubId = 1322201;
     name = "Yochai";
   };
+  yoctocell = {
```

Comment:
IIRC, adding yourself as a maintainer is preferred to be a separate commit.

Something like, `maintainers: add yoctocell`. Then apply the terminal-typeracer commit.